test/verify/check-networking-firewall

```diff
@@ -280,7 +313,47 @@ class TestFirewall(MachineCase):
         set_field("#tcp-ports", "80-90-100", "")
         check_error("Invalid range")
 
+    @skipImage("Filtering got fixed, and IDs changed to data-id in 184", "rhel-7-6-distropkg")
+    def testMultipleZones(self):
+        b = self.browser
+        m = self.machine
 
+        workSource = "192.168.1.0/24"
+        m.execute("firewall-cmd --permanent --zone=work --change-source={}".format(workSource))
+        m.execute("firewall-cmd --reload")
+
+        def addServiceToZone(service, zone):
+            b.click("caption button.btn-primary")
+            b.wait_present("#add-services-dialog")
```

Comment:
Unnecessary, click() and other actions wait for their selector to be present now.